### PR TITLE
Add UT for GetToken & SignOut apis

### PR DESCRIPTION
**Pull request by antrix1989**
## Proposed changes

Describe what this PR is trying to do.

## Type of change

- [ ] Feature work
- [ ] Bug fix
- [ ] Documentation
- [x] Engineering change
- [ ] Test
- [ ] Logging/Telemetry

## Risk

- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [ ] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [x] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

### CODE REVIEW

File: IdentityCore/src/broker_operation/response/browser_native_message_response/MSIDBrowserNativeMessageGetTokenResponse.m [hunk 1]
```diff
@@ -77,6 +77,11 @@ - (NSDictionary *)jsonDictionary
     response[@"account"] = accountJson;
     response[@"state"] = self.state;
     
+    __auto_type propertiesJson = [NSMutableDictionary new];
+    // TODO: once ests follow the latest protocol, this should be removed. Account ID should be read from accountJson.
```
Comment (kaisong1990) [inline IdentityCore/src/broker_operation/response/browser_native_message_response/MSIDBrowserNativeMessageGetTokenResponse.m:81]: nit: Please consider to add a PBI

File: IdentityCore/src/broker_operation/response/browser_native_message_response/MSIDBrowserNativeMessageGetTokenResponse.m [hunk 2]
```diff
@@ -77,6 +77,11 @@ - (NSDictionary *)jsonDictionary
     response[@"account"] = accountJson;
     response[@"state"] = self.state;
     
+    __auto_type propertiesJson = [NSMutableDictionary new];
```
Comment (kaisong1990) [inline IdentityCore/src/broker_operation/response/browser_native_message_response/MSIDBrowserNativeMessageGetTokenResponse.m:80]: Can this be a non-mutable dictionary?

**Review by kaisong1990 — APPROVED** [review]
Approved with a minor question

File: IdentityCore/src/broker_operation/response/browser_native_message_response/MSIDBrowserNativeMessageGetTokenResponse.m (hunk 2 above)
Comment (antrix1989) [reply IdentityCore/src/broker_operation/response/browser_native_message_response/MSIDBrowserNativeMessageGetTokenResponse.m:80]: If we make it non-mutable, we will need to check for nullability of idTokenObj.username, I prefer to leave it mutable to simplify code.

File: IdentityCore/src/broker_operation/response/browser_native_message_response/MSIDBrowserNativeMessageGetTokenResponse.m (hunk 1 above)
Comment (antrix1989) [reply IdentityCore/src/broker_operation/response/browser_native_message_response/MSIDBrowserNativeMessageGetTokenResponse.m:81]: https://identitydivision.visualstudio.com/Engineering/_boards/board/t/Auth%20Client%20-%20Apple/Backlog%20items/?workitem=2938432

**Pull request merged**